check-env.sh

```
@@ -0,0 +1,69 @@
+fi
+
+echo "✅ .env file exists"
+echo ""
+
+# Check required variables
+echo "📋 Required Cognito Variables:"
+echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+
+check_var() {
+    local var_name=$1
+    local var_value=$(grep "^${var_name}=" .env | cut -d'=' -f2-)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)